src/components/table-component/table.jsx

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
+
+/* 
+ @author [Kurtney J.](https://github.com/xhundo)
+*/
+
+// Table component build with named spaced format
+
+const StyledCell = styled.div`
+  width: 343px;
```

Comment:
Instead of giving these hard-coded widths, can we just add any necessary padding, maybe `min-height` instead of `height`, and then make it fill the available width of its container? I'm looking at the desktop views in figma and I want the table to just fill any available horizontal space wherever we put it

Reply:
What would seem to be an ideal padding for the cells, varies throughout all of them in figma:  `padding: 12px`?

Reply:
I've removed these hardcoded widths (shown in storybook shot above) and made these changes.